i18n/template.pot

```diff
@@ -114,19 +128,19 @@ msgstr ""
 msgid "Add BitPay Account?"
 msgstr ""
 
-msgid "Add Me"
+msgid "Add Encryption?"
 msgstr ""
 
-msgid "Add Recipient"
+msgid "Add Enryption"
```

Comment:
Please, review any typo and update the PR.

Reply:
There is a typo in `src/components/encrypt-password/encrypt-password.html`, I'll fix it and include it in this PR.